docker/maistra-builder_2.1.Dockerfile

```diff
@@ -32,6 +32,7 @@ ENV LICENSEE_VERSION=9.11.0
 ENV GOLANG_PROTOBUF_VERSION=v1.3.2
 ENV FPM_VERSION=1.11.0
 ENV SHELLCHECK_VERSION=v0.7.1
+ENV PROMU_VERSION=0.7.0
```

Comment:
I'm guessing this is the version that's supposed to work with the one we have in 2.1 branch

Reply:
Yeah, this is the one we're using for product.